src/ai/backend/storage/api/client.py

```diff
@@ -63,6 +64,7 @@ async def download(request: web.Request) -> web.StreamResponse:
     secret = ctx.local_config['storage-proxy']['secret']
     async with check_params(request, t.Dict({
         t.Key('token'): tx.JsonWebToken(secret=secret, inner_iv=download_token_data_iv),
+        t.Key('archive', default=False): t.ToBool | t.Null,
```

Comment:
Since we have a boolean default value, we don't have to put `| t.Null`.
For clarity, let's remove it.

Reply:
Updated.